opensfm/mesh.py

```diff
@@ -20,12 +20,23 @@ def triangle_mesh(shot_id, r, graph, data):
         return triangle_mesh_perspective(shot_id, r, graph)
     elif shot.camera.projection_type == 'fisheye':
         return triangle_mesh_fisheye(shot_id, r, graph)
+    elif shot.camera.projection_type == 'dual':
+        return triangle_mesh_dual(shot_id, r, graph)
     elif shot.camera.projection_type in ['equirectangular', 'spherical']:
         return triangle_mesh_equirectangular(shot_id, r, graph)
     else:
         raise NotImplementedError
 
 
+def triangle_mesh_dual(shot_id, r, graph):
+    shot = r.shots[shot_id]
+    cam = shot.camera
+    if cam.transition < 0.5:
```

Comment:
There is probably no reasonable threshold to decide this. The fisheye method will work for all camera types as long as their field of view is smaller than 180. So we could always use the fisheye method for simplicity. This way we will at least easily know which method was used.